### PR TITLE
Fix: --from-file and --files, auto remove duplicate files

### PR DESCRIPTION
**Pull request by pascalholthaus**
**What**: Auto remove duplicate files from --from-file and --files file list.

**Why**:

**How**:

**Checklist**:

- [ ] Tests
- [ ] Conventional commit message
- [ ] Documentation

### CODE REVIEW

**Review by y0urself — COMMENTED** [review]
Hacky hack! :)

**Review by y0urself — APPROVED** [review]

File: troubadix/troubadix.py [hunk 1]
```diff
@@ -97,10 +97,17 @@ def generate_patterns(
...
+        return list(
+            set(
+                [
+                    Path(f)
+                    for f in include_file.read_text(
+                        encoding="utf-8"
+                    ).splitlines()
+                ]
+            )
+        )
```
Comment (bjoernricks) [inline troubadix/troubadix.py:110]: Just return the set here. A set is also an Iterable

**Review by bjoernricks — CHANGES_REQUESTED** [review]

File: troubadix/troubadix.py (hunk 1 above)
Comment (pascalholthaus) [reply troubadix/troubadix.py:110]: I get some errors with it.
  File "/home/work/work/troubadix/troubadix/troubadix.py", line 162, in main
    first_file = files[0].resolve()
TypeError: 'set' object is not subscriptable

Comment (bjoernricks) [reply troubadix/troubadix.py:110]: Ok in that case the typing of the return value of the `from_file` function should be changed to `List` too

Comment (pascalholthaus) [reply troubadix/troubadix.py:110]: Ah sorry missed it.

**Review by bjoernricks — DISMISSED** [review]

**Pull request closed without merging**